### PR TITLE
Inventory Handlers

### PR DESCRIPTION
**Pull request by jadbalout**
Added inventory handlers: qpa, qpp, ai, gi

### CODE REVIEW

File: as2/server/inventory.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+### Add Item
+Sent to clients after an item gets added to their penguin's inventory.
+
+`ai`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|itemID|Unsigned integer|No|
+|coins|Unsigned integer|No|
```
Comment (widd) [inline as2/server/inventory.md]: coins -> newCoins

File: as2/server/inventory.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|itemID|Unsigned integer|Yes|
+
+##### Example
+`%xt%gi%0%413%414%567%`
+
+### Add Item
+Sent to clients after an item gets added to their penguin's inventory.
```
Comment (widd) [inline as2/server/inventory.md:59]: Add a note that the penguin's new coin balance is sent in this message for items that have costs.

**Review by widd — CHANGES_REQUESTED** [review]
Looks good, just need some extra detail for Add Item.

**Comment by jadbalout** [comment]
Done

**Review by widd — APPROVED** [review]
Looks good.

**Pull request merged**